recipes/libjxl/all/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8.11)
```

```suggestion
cmake_minimum_required(VERSION 3.1)
```

TARGETS requires cmake 3.1 at least